lib/autokey/configmanager.py

```diff
@@ -19,26 +19,19 @@
 import os.path
 import shutil
 import logging
-import pickle
 import glob
 import threading
-import subprocess
+import re
 
-from . import common, monitor
+from . import common
 from .iomediator_constants import X_RECORD_INTERFACE
 
-try:
```

Comment:
This is a compatibility fix for Python 2.4-2.6. just import the standard library json module.